content/en/docs/instrumentation/js/exporters.md

```diff
@@ -59,6 +73,9 @@ const {
 const {
   OTLPTraceExporter,
 } = require("@opentelemetry/exporter-trace-otlp-http");
+const {
+  OTLPMetricExporter
+} = require("@opentelemetry/exporter-metrics-otlp-http");
```

Comment:
same here with the exporter package :slightly_smiling_face:

Reply:
done